src/UI/Seller/src/app/products/components/product-edit/product-edit.component.ts

```diff
@@ -834,12 +853,16 @@ export class ProductEditComponent implements OnInit, OnDestroy {
     if (superHSProduct.PriceSchedule.PriceBreaks[0].Price === null)
       superHSProduct.PriceSchedule.PriceBreaks[0].Price = 0
     if (this.imageFiles.length > 0) {
-      const imgAssets = await this.assetService.uploadImageFiles(this.imageFiles);
-      superHSProduct.Product.xp.Images = imgAssets
+      const imgAssets = await this.assetService.uploadImageFiles(
+        this.imageFiles
+      )
+      ;(superHSProduct.Product.xp as any).Images = imgAssets
```

Comment:
Can you update the type for `SuperHSProduct` so that you don't need to cast it as type any? By casting it to any we are defeating the purpose of using a strongly typed system

Reply:
I updated the line.  I don't believe this was something I actively changed to begin with.